clearblade.md

```diff
@@ -1328,21 +1334,19 @@ This function does not return query results.
 
 ## ClearBlade.Database.performOperation(callback, argument)
 
-This function takes a callback as the first argument and a variable number of arguments after the callback.
+This function takes a callback as the first argument and a variable number of string arguments after the callback.
 
 * @param {function} callback - Function that handles the response from the server
-* @param {string} param1 - Commands that are used in the external databases.
-* @param {string} param2 - Commands that are used in the external databases.
-* @param {string} param3 - Commands that are used in the external databases.
+* @param {...string} arguments - Arguments that are used in the external databases.
 
-
-|Functions| sql | mongodb | couchdb |
-|:---|:---|:---|:---|
-|param1|query, ex: "SELECT * from myTable where name='Bob'" |dbCommand, ex: "find"|httpMethod, ex: "POST"|
-|param2| - | - | uri, ex: "/mydb/bulk_docs"|
-|param3| - | - |data, ex: "{\"docs\": [{\"name\": \"Bob\", \"age\": 100}]}"|
+We define how the param `arguments` looks like for each database below.
 
 ### MongoDB 
+
+- arguments
+  - dbCommand: one of the below supported collection methods
+
```

Comment:
Add the link to the MongoDB cli commands here - https://docs.mongodb.com/manual/reference/method/js-collection/